docs/_articles/en/apex/apex-get-started.md

```diff
@@ -0,0 +1,108 @@
+2. Run the `SFDX: Refresh SObject Definitions` command from the Command Palette to get completion suggestions for your SObjects related code.
+3. Scroll down to Custom Object in the Org Browser and locate the `Book_c` object. Click the retrieve icon to run `SFDX: Retrieve Source from Org`.
+4. From the command palette run `SFDX:Create Apex Class` and create a class called `MyHelloWorld`. 
+5. Add a method called `applyDiscount` to this class. 
+   
+We’ll make this method both public and static. Because it’s a static method, you don't need to create an instance of the class to access the method—you can just use the name of the class followed by a dot (.) and the name of the method.
+
+This method takes one parameter, a list of Book records, which is assigned to the variable books. The method iterates through a list of books and applies a 10% discount to the current book price:
+
+```
+public with sharing class MyHelloWorld {
+    public MyHelloWorld() {
+
+    }
+    public static void applyDiscount(Book__c[] books) {
+        for (Book__c b :books){
+           b.Price__c *= 0.9;
+        }
+     }
+}
+```
+Next we’ll add a trigger that calls this `applyDiscount` method. 
+
+### Add an Apex Trigger
+An Apex trigger is a piece of code that executes before or after records of a particular type are inserted, updated, or deleted from the Lightning Platform database. Every trigger runs with a set of context variables that provide access to the records that caused the trigger to fire.
+1. From the command palette run SFDX:Create Apex Trigger and create a new class called HelloWorldTrigger. 
+2. Update the default template to this trigger definition:
+
+```
+trigger HelloWorldTrigger on Book__c (before insert) {
+
+   Book__c[] books = Trigger.new;
+
+   MyHelloWorld.applyDiscount(books);
+}
+```
+
+We now have the code that is needed to update the price of all books that get inserted. Next, let’s add a test class and add a unit test. Unit tests are an important part of writing code and are required.
+### Write a Unit Test
+Now we'll add a test class with one test method. We’ll also run the test and verify code coverage. The test method exercises and validates the code in the trigger and class. Also, it enables you to reach 100% code coverage for the trigger and class.
+For our example, we’ll create a test class that inserts a new book object which sets off the Apex trigger we wrote earlier. 
+To create the test class:
+
+1. From the command palette run SFDX:Create Apex Class and create a class called HelloWorldTestClass. 
+2. Paste the following code in the HelloworldTestClass.cls file:
+
+```
+@isTest 
+private class HelloWorldTestClass {
+    static testMethod void validateHelloWorld() {
+       Book__c b = new Book__c(Name='Behind the Cloud', Price__c=100);
+       // Insert book
+       insert b;    
+       // Retrieve the new book
+       b = [SELECT Price__c FROM Book__c WHERE Id =:b.Id];
```

Comment:
As a best practice, I'd create a new Book__c object rather than overwrite the one in memory.

Reply:
@sbudhirajadoc wanted to see your thoughts on this suggestion